src/localization/en.js

```diff
@@ -116,15 +116,27 @@ export default {
       predictions:     'Predictions',
     },
 
+    formHelpers: {
+      type:    ' Type',
```

Comment:
@ValerieKenyon : Could you undo the changes you made to parts that weren't in `currentBenefits`? You can just re-type them as you currently see them in the dev branch.

Reply:
Just did that.  Hope I did it okay.